tests/rptest/e2e_tests/flink_scale_test.py

```diff
@@ -88,12 +88,12 @@ def assert_failed_jobs(job_list):
                 "Flink reports fails for " \
                 f"transaction workload:\n{desc}"
 
-        def get_hwm_for_topic(topic):
-            _hwm = 0
+        def get_total_events_for_topic(topic):
```

Comment:
tidy up the commit history? This commit probably should be squashed among the previous commits.

Reply:
Done